src/shared_modules/utils/windowsHelper.h

```diff
@@ -245,6 +245,18 @@ namespace Utils
         return serialNumber;
     }
 
+    static std::string buildTimestamp(const ULONGLONG time)
+    {
+        // Format of value is 18-digit LDAP/FILETIME timestamps.
+        // 18-digit LDAP/FILETIME timestamps -> Epoch/Unix time
+        // (value/10000000ULL) - 11644473600ULL
+        const time_t epochTIme { static_cast<long int> ((time / 10000000ULL) - WINDOWS_UNIX_EPOCH_DIFF_SECONDS) };
```

Comment:
Typo: `epochTIme` instead of `epochTime`

Reply:
This was an old PR that was approved before, I noticed this typo and changed it here https://github.com/wazuh/wazuh/pull/14161
I fixed it here too. This branch needed a rebase anyway.